llvm/test/CodeGen/AMDGPU/sdivrem24.ll

```diff
@@ -258,7 +258,7 @@ define amdgpu_kernel void @no_srem25_i24_i25_i32(ptr addrspace(1) %out, ptr addr
 }
 
 ; FUNC-LABEL: {{^}}no_srem25_i25_i24_i32:
-; SI-NOT: v_cvt_f32_i32
```

Comment:
changing -NOT checks makes me nervous. make this looser to check both types of conversions?

Reply:
The new generated assembly was now always generating _i32 version but not generating _u32. Could you clarify by 'looser'?

Reply:
reverted with last commit incrementing AtLeast